app/views/layouts/application.html.erb

```diff
@@ -12,8 +12,14 @@
   </head>
 
   <body>
-    <nav class="crumbs">
-    </nav>
+    <% unless current_page?(sign_in_path) %>
+      <nav class="crumbs">
+          <ul>
+              <li class="crumb"><a href="#"><%= t(".title") %></a></li>
+              <li class="crumb"><%= link_to t(".sign_in"), sign_in_path %></li>
+          </ul>
+      </nav>
+    <% end %>
```

Comment:
**suggestion:** Are you sure you want to hide the entire `nav` element while on the "Sign in" page? The other link is to your home page. How does the page look when you hide these links?

Reply:
See attached photo for how it looks like with the links hidden
![Screenshot 2022-12-08 at 17 09 04](https://user-images.githubusercontent.com/3297508/206467111-847c2c09-fb2f-4833-a469-03bc023c51e0.jpg)

At the moment, the home page is: `articles#new ` 
In order to view the homepage, the user has to be signed in.